src/admin.go

```diff
@@ -244,10 +647,22 @@ func Create(options Options) {
 		cfSession = openCloudFront(options.AWSKey, options.AWSSecret)
 	}
 
-	_, err := exec.LookPath("aws")
+	//official sdk connection
+	if awsSession == nil {
+		awsSession = session.New(&amazonaws.Config{
+			Region:      amazonaws.String(options.AWSRegion),
+			Credentials: credentials.NewStaticCredentials(options.AWSKey, options.AWSSecret, ""),
```

Comment:
This will still pull the creds from the AWS config file (if they're not specified on the command line)?